python/golem_task_api/client.py

```diff
@@ -45,7 +45,7 @@ def running(self) -> bool:
         pass
 
     @abc.abstractmethod
-    def start(self, command: str, port: int) -> Tuple[str, int]:
+    async def start(self, command: str, port: int) -> Tuple[str, int]:
```

Comment:
While conceptually this is fine making this method async makes it a little bit prone to spawning the service process in the same asyncio loop as the spawner - which is *not* what we want here. I.e. after adjusting blenderapp tests it might be tempting to just call `await main` here https://github.com/golemfactory/blenderapp/blob/master/tests/test_commands.py#L34 which will not work fine. Maybe just add a comment in the docstring that the process should not be spawned in the same event loop.

Reply:
You mean that implementor of this method should not use the event loop room which it was called to actually run the service? I didn't even think someone might do this but I can add a comment.